doc/python/configuration-options.md

```diff
@@ -132,7 +298,52 @@ fig = go.Figure(go.Bar(
 
 fig.update_layout(xaxis = {'type': 'date'})
 
-fig.show(config = {'doubleClickDelay': 1000})
+fig.show(config=config)
+```
+
+#### Configuring Figures in Dash Apps
+
+The same configuration dictionary that you pass to the `config` parameter of the `show()` method can also be passed to the `config` parameter of a `dcc.Graph` component.
+
+```python
+import dash
+import dash_core_components as dcc
```

Comment:
you need to add dcc and html to the `requirements.txt` in the `doc` and `binder` directories

Reply:
Ah, that makes sense. Thanks!